low-level/files/opt/seagate/sspl/bin/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -325,8 +328,27 @@ def sspl_bundle_cleanup():
     if os.path.exists(sspl_bundle_tmp):
         shutil.rmtree(sspl_bundle_tmp)
 
+def validate_sb_request():
+    valid_request = True
+    if os.path.isfile(onGoingRequestPath):
+        last_reboot = int(psutil.boot_time())
+        req_start_time = int(pathlib.Path(onGoingRequestPath).stat().st_mtime)
+        current_time = int(time.time())
+        # Check if system is rebooted in between or request is expired.
+        # If Yes, Then clear existing request cache.
+        is_req_expired = (current_time - req_start_time) > SB_REQUEST_TIMEOUT
+        if last_reboot > req_start_time or is_req_expired:
+            valid_request = True
```

Comment:
dont need to make it 'true' again here at line 341 & 346, given default is true

Reply:
Or `valid_request` can be initialized as `False`.

Reply:
Done